include/zigbee/zigbee_error_handler.rst

```diff
@@ -0,0 +1,31 @@
+.. _lib_zigbee_error_handler:
+
+Zigbee error handler
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Zigbee error handler library provides error codes for ZBOSS functions that return codes other than ``RET_OK``.
+It is used by default in the :ref:`zigbee_ug_logging_stack_logs` when you enable :ref:`zigbee_ug_logging`.
+Enabling this library allows you to use the error handling utilities for the :ref:`nrfxlib:zboss` stack when working with the :ref:`ug_zigbee` protocol in the |NCS|.
+
+.. _lib_zigbee_error_handler_options:
+
+Configuration
+*************
+
+You can configure the Zigbee error handler library using the :option:`CONFIG_ZBOSS_ERROR_PRINT_TO_LOG` Kconfig option.
+This option is enabled by default when you enable :ref:`zigbee_ug_logging` using the :option:`CONFIG_LOG`.
+The :option:`CONFIG_ZBOSS_ERROR_PRINT_TO_LOG` option automatically enables the :option:`CONFIG_ZIGBEE_ERROR_TO_STRING_ENABLED` Kconfig option, which obtains the ZBOSS error code that is supplied and printed to the log.
+
+API documentation
+*****************
+
+| Header file: :file:`include/zigbee/zigbee_error_handler.h`
+| Source file: n/a
```

Comment:
Is this needed when empty?
Also, the `zb_error_to_string.c` in nrfxlib is a source file used by this API.

Reply:
OK, so I guess we need to describe in the overview that this library has the ZBOSS source file in nrfxlib, no? Not sure how many other libs have sources in a different submodule...

Reply:
AFAIK the `zb_error_to_string.c` i not part of the ZBOSS stack.
@tomchy Do you remember if the `zb_error_to_string.c` needs to be in the `nrxlib`? Maybe we can move it to `nrf`.